task.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+# создаём каталог task с вложенными директориями
+# task
+#   dir1
+#   dir2
+#   dir3
+#       dir4
+mkdir -p task/dir1
```

Comment:
Можно в одну команду
```suggestion
mkdir -p task/dir1 task/dir2 task/dir3/dir4
```